examples/mini-farmer.rs

```diff
@@ -81,36 +93,75 @@ async fn main() -> anyhow::Result<()> {
 
     let node_dir = base_path.join("node");
     let node = match chain {
-        Chain::Gemini3D => Node::gemini_3d().dsn(
+        Chain::Gemini3d => Node::gemini_3d().dsn(
             subspace_sdk::node::DsnBuilder::gemini_3d()
                 .provider_storage_path(node_dir.join("provider_storage")),
         ),
         Chain::Devnet => Node::devnet().dsn(
             subspace_sdk::node::DsnBuilder::devnet()
                 .provider_storage_path(node_dir.join("provider_storage")),
         ),
+        Chain::Dev => Node::dev().dsn(
+            subspace_sdk::node::DsnBuilder::dev()
+                .provider_storage_path(node_dir.join("provider_storage")),
+        ),
     }
     .role(node::Role::Authority);
 
     #[cfg(feature = "executor")]
-    let node = if executor {
-        node.system_domain(subspace_sdk::node::domains::ConfigBuilder::new())
-    } else {
-        node
+    let node = match domain {
+        Some(domain) => node.system_domain({
+            use subspace_sdk::node::domains;
+
+            let system_domain = domains::ConfigBuilder::new()
+                .rpc(
+                    subspace_sdk::node::RpcBuilder::new()
+                        .http("127.0.0.1:9990".parse().unwrap())
+                        .ws("127.0.0.1:9991".parse().unwrap()),
+                )
+                .role(node::Role::Authority);
+            let rpc = subspace_sdk::node::RpcBuilder::new()
```

Comment:
Why do all of the domains share the same rpc?

Reply:
I can make different rpcs for different domains, but currently this `mini-farmer` runs only one domain